docs/src/sway-program-types/libraries.md

```diff
@@ -4,7 +4,7 @@ Libraries in Sway are files used to define new common behavior. An example of th
 
 ## Writing Libraries
 
-Functions in libraries can also read from storage and interact with the state. Libraries are denoted using the `library` keyword at the beginning of the file, followed by a name so that they can be imported. E.g. `library foo;`.
+Libraries are denoted using the `library` keyword at the beginning of the file, followed by a name so that they can be imported. e.g. `library foo;`.
```

Comment:
```suggestion
Libraries are denoted using the `library` keyword at the beginning of the file, followed by a name so that they can be imported, e.g. `library foo;`.
```
`E.g.` should either be capitalized as it is at the beginning of the sentence, or the previous sentence should actually be a clause ending in a comma. 
(source)[https://ell.stackexchange.com/questions/56861/capitalize-i-e-and-e-g]